### DISCUSSION TITLE
memorabilia

### DISCUSSION THREAD

**Discussion by rejetto** [rejetto/hfs#384] [Show and tell]
this is what HFS was like in 2003 (exe + sources)
[hfs06.zip](https://github.com/rejetto/hfs/files/13178475/hfs06.zip)

and this in 2002
[hfs02.zip](https://github.com/rejetto/hfs/files/13178484/hfs02.zip)

**Comment by W-i-n-7** [comment]
wait thats 2022?

**Reply by rejetto** [reply]
typo.... 2002
